src/pattern_matching/mod.rs

```diff
@@ -14,8 +14,8 @@
 //! * KMP algorithm: the classical ancestor.
 //! * Ukkonens algorithm: approximate pattern matching with dynamic programming.
 //! * Myers algorithm: linear-time approximate pattern matching with edit distance for small patterns
-//!
-//! Another fast pattern matching algorithm is available in the twoway crate: https://crates.io/crates/twoway
+//! 
+//! This library provides heavily optimized routines for string search primitives. Available at: https://crates.io/crates/memchr
```

Comment:
I would suggest retaining the structure of the original comment, for example:

Another library that provides heavily optimized routines for string search primitives is memchr: https://crates.io/crates/memchr